crypto/rand/drbg_lib.c

```diff
@@ -169,13 +181,13 @@ RAND_DRBG *RAND_DRBG_new(int type, unsigned int flags, RAND_DRBG *parent)
  */
 void RAND_DRBG_free(RAND_DRBG *drbg)
 {
-    if (drbg == NULL)
-        return;
-
-    if (drbg->meth != NULL)
-        drbg->meth->uninstantiate(drbg);
-    CRYPTO_free_ex_data(CRYPTO_EX_INDEX_DRBG, drbg, &drbg->ex_data);
-    OPENSSL_clear_free(drbg, sizeof(*drbg));
+    if (drbg != NULL) {
```

Comment:
I kinda prefer the old test/return style.  It's how almost all of our other free functions are written.

Reply:
Fixed.